pcsx2/CDVD/IsoFS/IsoFS.cpp

```diff
@@ -49,8 +50,9 @@ IsoDirectory::IsoDirectory(SectorSource& r)
 	uint i = 16;
 
 	m_fstype = FStype_ISO9660;
-
-	while( !done )
+	
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	while( !done && loadFromDisc)
```

Comment:
This isn't the right solution for suppressing the "invalid partition descriptor" message. It doesn't make sense to me.